modules/ocf_filehost/templates/exports.ebb

```diff
@@ -7,3 +7,7 @@
            dev-ssh(rw,fsid=0,no_subtree_check,no_root_squash) \
            apphost(rw,fsid=0,no_subtree_check,no_root_squash) \
            dev-apphost(rw,fsid=0,no_subtree_check,no_root_squash)
+
+<% $kubernetes_workers = lookup('kubernetes::worker_nodes') -%>
+<% $worker_exports = $kubernetes_workers.map |$worker| { "${worker}(rw,fsid=0)" } -%>
+/opt/homes/services/mastodon <%= $worker_exports.join(" ") %>
```

Comment:
might want to `.join("\\ \n")` (that is probably wrong, but you get the point) so the exports file is human readable if we have a bunch of workers.